spec/models/item_spec.rb

```diff
@@ -97,6 +97,21 @@
       expect(Item.least_popular(5)).to eq([tire3, tire4, tire5, tire6, tire1])
     end
 
+    after(:each) do
```

Comment:
does this need to be in here twice? looks like the same thing is done again on lines 146-150